src/registrar/models/domain_application.py

```diff
@@ -671,7 +676,7 @@ def action_needed(self):
         ],
         target=ApplicationStatus.APPROVED,
     )
-    def approve(self):
+    def approve(self, do_fake_send_email=False):
```

Comment:
I'm wondering if instead of having a flag for fake sending email that doesn't get sent, we make the flag indicate that an email **is** sent -- that way when we make tests approving emails, we can set all tests to set the flag as false and indicate that we are setting send_email as false because we don't want our tests to actually send emails? Or in our tests, we can have a function like `test_approve` that performs an email approval with the send_email flag set to false that we can call in our tests

Reply:
I'm not familiar with how emails work, so feel free to not go with this suggestion if it doesn't align with how the registrar sends and tests emails!

Reply:
Yeah I have no problem making that change, it would be more clear. I agree that setting it to `send_email=True` makes more contextual sense